thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -331,15 +331,17 @@ public Response createAnomalyFunction(@NotNull @QueryParam("dataset") String dat
       break;
     case HOURS:
     default:
-      windowDelayTime = 2;
+      windowDelayTime = 0;
       windowDelayTimeUnit = TimeUnit.HOURS;
     }
     anomalyFunctionSpec.setWindowDelayUnit(windowDelayTimeUnit);
     anomalyFunctionSpec.setWindowDelay(windowDelayTime);
 
     // setup detection frequency if it's minutely function
+    // the default frequency is determined by data granularity
+    // this frequency is a override to lower the system overload
     if (dataGranularity.getUnit().equals(TimeUnit.MINUTES)) {
-      TimeGranularity frequency = new TimeGranularity(30, TimeUnit.MINUTES);
```

Comment:
in graph is real time. All data is streaming to ingraph database through kafka. If there is delay (more than say 5 minutes, it will be a big production issue). 

On the other side, our sign test method do not sensitive to missing data. Considering we are doing 6 hour moving window. We can tolerate the delay. This frequency change from 30 minutes to 15 minutes is not significant: that 6 new data points vs 3 data points.  But if we change granularity to 1 minutes in future. 15 minutes means 15 new observations -- this can really move the needle. And we may consider further push this to 5 minutues, if we really targeting for MTTD for some special use cases.